src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier{
```

Comment:
The `Lottery` class should not extend `ColorSupplier`. Inheritance is not appropriate here because `Lottery` is not a specialized form of `ColorSupplier`. Consider using composition instead by creating an instance of `ColorSupplier` within `Lottery` to get a random color.